### DISCUSSION TITLE
Using existing openebs disk volumes with new openebs installation

### DISCUSSION THREAD

**Discussion by k8sify** [openebs/openebs#3657] [Q&A] [answered]
Hi,

I'd like to recover the data on my existing openebs install deployed on a multi node Microk8s cluster. I'm using cstor `storageClass` backed by `CStorPoolCluster` using blockDevices. The cluster control plane is corrupted so I cant run any application level backups on the PVCs so I'm thinking of using `dd` to image the disks with a look to restoring them. I wondered if anyone knew if it's possible to launch a fresh openebs install using those existing disks?

Example dd command:

```
 dd if=/dev/sdc1 bs=4M | gzip | ssh node@10.1.2.3 'cat > /mnt/md0/node2/sdc1.img.gz'
```

Any help would be very welcome!

**Comment by avishnu** [comment, marked as answer]
Hope you were able to recover your data !!
Thanks for being a user of OpenEBS !!
We'd like to suggest Mayastor as the alternative to CStor.
Mayastor is a replicated storage solution (just like CStor) but it's very fast as compared to CStor.
Mayastor is under active development and support, unlike CStor.

https://github.com/openebs/mayastor
https://mayastor.gitbook.io/introduction/
